ols/utils/config.py

```diff
@@ -1,3 +1,6 @@
+# we are disabling ruff check for long lines because of the commented
+# code at the bottom
+# ruff: noqa: E501
```

Comment:
@syedriko please don't forget to remove this if your PR is merged after this one.

cc @bparees